src/Layout/Presentation/SingleDistrict/SingleDistrict.tsx

```diff
@@ -152,30 +158,34 @@ export class SingleDistrict extends React.Component<SingleDistrictProps, {}> {
                                         </span>
                                     );
                                 }
-                                return row.value;
+                                return value;
                             },
                             show: calculateVulnerable,
                         },
                         {
                             id: "lastSeatQuotient",
-                            Header: "Siste kvotient",
+                            Header: <span className="is-pulled-right" >Siste kvotient</span>,
                             accessor: (d: PartyResult) =>
-                                d.votes > 0 && quotientMap ? quotientMap.get(d.partyCode)!.toFixed(decimals) : null,
+                                d.votes > 0 && quotientMap ? quotientMap.get(d.partyCode): null,
                             Cell: (row) => {
+                                const value = row.value ? replaceComma(row.value.toFixed(decimals)) : row.value;
```

Comment:
How come you are just replacing the comma instead of using the numberFormatFraction function you used before?

Reply:
I did not want to separate the thousands for 'Siste kvotient'. Design choice. Same as in remainderQuotients.